web/libs/frontend-test/src/helpers/LSF/Relations.ts

```diff
@@ -1,15 +1,53 @@
 export const Relations = {
+  get relations() {
+    return cy.get('.lsf-relations');
+  },
+  get relationOrderList() {
+    const relationList = [];
+
+    cy.get('.lsf-relations__item').each(($el) => {
+      const from = $el.find('.lsf-detailed-region .lsf-labels-list span').first().text().trim();
+      const to = $el.find('.lsf-detailed-region .lsf-labels-list span').last().text().trim();
+      relationList.push({ from, to });
+    });
+
+    return cy.wrap(relationList);
+  },
+  get hideAllRelationsButton() {
+    return cy.get('[aria-label="Hide all relations"]');
+  },
+  get showAllRelationsButton() {
+    return cy.get('[aria-label="Show all relations"]');
+  },
+  get ascendingOrderRelationButton() {
+    return cy.get('[aria-label="Sort by ascending"]');
+  },
+  get descendingOrderRelationButton() {
+    return cy.get('[aria-label="Sort by descending"]');
+  },
+  get hiddenRelations() {
+    return this.relations
+      .should('be.visible')
+      .get('.lsf-relations__item_hidden .lsf-relations__content');
+  },
   hasRelations(count: number) {
     cy.get('.lsf-details__section-head').should('have.text', `Relations (${count})`);
   },
   hasRelation(from: string, to: string) {
     cy.get('.lsf-relations').contains(from).closest('.lsf-relations').contains(to);
   },
+  hasHiddenRelations(count: number) {
+    this.hiddenRelations.should('have.length', count);
+  },
   toggleCreation() {
     cy.get('.lsf-region-actions__group_align_left > :nth-child(1) > .lsf-button__icon').click();
   },
-  toggleCreationWithHotkey() {
-    // hotkey is alt + r
-    cy.get('body').type('{alt}r');
-  },
```

Comment:
Why was this helper method removed? I just did a quick search and it is still used elsewhere in the codebase.

Reply:
Apologies for the oversight. I've restored the accidentally deleted helper method.